python/tests/test_property_graph.py

```diff
@@ -18,6 +18,38 @@ def test_load(property_graph):
     assert len(property_graph.edge_schema()) == 19
 
 
+def test_write(property_graph):
+    # TODO(amp): mark_all_properties_persistent shouldn't be required. Why is it?
+    property_graph.mark_all_properties_persistent()
+    with TemporaryDirectory() as tmpdir:
+        property_graph.write(tmpdir)
+        old_property_graph = property_graph
+        del property_graph
+        property_graph = PropertyGraph(tmpdir)
+    assert property_graph.num_nodes() == 29092
+    assert property_graph.num_edges() == 39283
+    assert len(property_graph.node_schema()) == 31
+    assert len(property_graph.edge_schema()) == 19
+
+    assert property_graph == old_property_graph
+
+
+# TODO(amp): Reinstant this test once it matches the actual RDG semantics.
```

Comment:
Maybe we can invite a reviewer who worked on `PropertyGraph`?

Reply:
That is weird, especially if you are interfacing with the `PropertyGraph `directly and not a variant like `AttributedGraph`.  Actually, now that I'm thinking about it, this problem happened when I was loading a `DistGraph`.

Reply:
> Actually, now that I'm thinking about it, this problem happened when I was loading a `DistGraph`.

Interesting. I will leave this test disabled and let you create an issue for this problem if needed. It doesn't seem like a fatal issue at the moment, though it is something we should address eventually.